From cc50b04b66cfe036e599efc5c69f4d8a081f91e2 Mon Sep 17 00:00:00 2001
From: NelloKudo <marshnelloosu@gmail.com>
Date: Sat, 13 Dec 2025 16:28:09 +0100
Subject: [PATCH] ntdll: change logging and redirect to Spritz repository

---
 dlls/ntdll/loader.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/dlls/ntdll/loader.c b/dlls/ntdll/loader.c
index 9a757709980..be6ce574dd6 100644
--- a/dlls/ntdll/loader.c
+++ b/dlls/ntdll/loader.c
@@ -4672,7 +4672,10 @@ void loader_init( CONTEXT *context, void **entry )
                     MESSAGE("ntsync is explicitly disabled.\n");
             }
             if (FIXME_ON(winediag))
-                MESSAGE("winediag: this wine %s contains many experimental patches, please don't report bugs to winehq.org.\n", wine_get_version());
+            {
+                FIXME_(winediag)("Spritz-Wine-TkG (staging) %s is a downstream version containing specific patches for games.\n", wine_get_version());
+                FIXME_(winediag)("Please don't report bugs about it on winehq.org or at wine-tkg but use https://github.com/NelloKudo/Wine-Builds instead.\n");
+            }
         }
         NtClose( startup_event );
     }
-- 
2.52.0

