From d4e4f8214896fdd8af5eee434a07f751899217dc Mon Sep 17 00:00:00 2001
From: mkrsym1 <mkrsym1@gmail.com>
Date: Mon, 8 Sep 2025 21:05:27 +0200
Subject: [PATCH 08/10] loader: Copy arial.ttf to fonts

---
 loader/wine.inf.in | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 61acfba64a4..50adce9ba4d 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -54,7 +54,7 @@ AddService=MountMgr,0x800,MountMgrService
 RegisterDlls=RegisterDllsSection
 WineFakeDlls=FakeDlls
 UpdateInis=SystemIni
-CopyFiles=ColorFiles,EtcFiles,InfFiles,NlsFiles,SortFiles
+CopyFiles=ColorFiles,EtcFiles,InfFiles,NlsFiles,SortFiles,FontFiles
 AddReg=\
     Classes,\
     ContentIndex,\
@@ -112,7 +112,7 @@ AddReg=\
 [BaseWow64Install]
 RegisterDlls=RegisterDllsSection
 WineFakeDlls=FakeDllsWin32,FakeDllsWow64
-CopyFiles=NlsFiles
+CopyFiles=NlsFiles,FontFiles
 AddReg=\
     CurrentVersion,\
     Debugger,\
@@ -1544,9 +1544,13 @@ normnfkd.nls
 [SortFiles]
 sortdefault.nls
 
+[FontFiles]
+arial.ttf
+
 [WineSourceDirs]
 NlsFiles  = nls
 SortFiles = nls
+FontFiles = fonts
 
 [DestinationDirs]
 ColorFiles = 23
@@ -1554,3 +1558,4 @@ EtcFiles  = 12,etc
 InfFiles  = 17
 NlsFiles  = 11
 SortFiles = 10,globalization\sorting
+FontFiles = 20
-- 
2.52.0

